index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+<!-- Joke easter egg:
+ Q - Why do Java programmers wear glasses?
+ A - They can’t C#. :D
+ -->
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <title>Photo gallery</title>
+
+    <!-- Links for the bootstrap -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-QWTKZyjpPEjISv5WaRU9OFeRpok6YctnYmDr5pNlyT2bRjXh0JMhjY6hW+ALEwIH" crossorigin="anonymous">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/js/bootstrap.bundle.min.js"
+        integrity="sha384-YvpcrYf0tY3lHB60NNkmXc5s9fDVZLESaAA55NDzOxhy9GkcIdslK1eN7N6jIeHz"
+        crossorigin="anonymous"></script>
+
+    <!-- A script for the JavaScript -->
+    <script defer src="assets/js/app.js"></script>
+
+    <!--A link for the CSS -->
+    <link rel="stylesheet" href="assets/css/style.css">
+</head>
+
+<body>
+    <div class="container mt-5">
```

Comment:
I assume you missed it by I asked not to use `bootstrap` for grid, notice for next time